src/kernel-install/50-depmod.install

```diff
@@ -23,6 +23,8 @@ set -e
 COMMAND="${1:?}"
 KERNEL_VERSION="${2:?}"
 
+[ -w "/lib/modules" ] || exit 0
```

Comment:
this makes sense in any case, can you split this out, we can merge this ahead of the rest.

Reply:
https://github.com/systemd/systemd/pull/25940